aspnetcore/getting-started/index.md

```diff
@@ -4,7 +4,7 @@ author: rick-anderson
 description: A short tutorial that creates and runs a basic Hello World app using ASP.NET Core. 
 ms.author: riande
 ms.custom: mvc
-ms.date: 09/22/2019
+ms.date: 01/07/2020
```

Comment:
ms.date: 1/7/2020
```suggestion
ms.date: 01/07/2020
```

Reply:
Thanks.  Commit suggestion available but still grayed out.   I will set to ready for review just to see if that makes any difference.

Reply:
Still grayed out.

Reply:
Can you unresolve this so GitHub can see it?

Reply:
unresolved.